src/styles/Recipe.css

```diff
@@ -0,0 +1,18 @@
+.card-title {
+  background-color: transparent;
+  background-repeat: no-repeat;
+  border: none;
+  overflow: hidden;
+  outline: none;
+}
+
+.modal-box {
+  border-style: ridge;
+  outline: black;
+  background-color: transparent;
+}
+
+.modalTitle {
```

Comment:
nit: be consistent with your naming. everything else is -notation, so here should be modal-title